packages/framework/test-client-utils/src/ConsoleLogger.ts

```diff
@@ -0,0 +1,17 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { ITelemetryBaseLogger, ITelemetryBaseEvent } from "@fluidframework/common-definitions";
+
+// Basic implementation of an ITelemetry Logger. This logger will be passed into a client constructor
+// and gets hooked up to the container telemetry system.
+export class ConsoleLogger implements ITelemetryBaseLogger {
+    constructor() {}
+
+    send(event: ITelemetryBaseEvent) {
+        event.Event_Time = Date.now();
```

Comment:
Is this an expected pattern that the listener log the time of the event?

Reply:
no. usually this done by the underlying system being logged into